simulator/main.go

```diff
@@ -9,60 +9,138 @@ package main
 import (
 	"flag"
 	"fmt"
+	"log"
 	"os"
 	"os/signal"
+	"strconv"
+	"sync"
 	"syscall"
 
 	"github.com/goburrow/serial"
 	"github.com/tbrandon/mbserver"
 )
 
 const (
-	ProtocolTCP = "TCP"
-	ProtocolRTU = "RTU"
-	ModbusPort  = "1502"
+	protocolTCP            = "TCP"
+	protocolRTU            = "RTU"
+	defaultDevicePort      = 1502
+	host                   = "0.0.0.0"
+	startingPortEnvName    = "STARTING_PORT"
+	simulatorNumberEnvName = "SIMULATOR_NUMBER"
+	defaultStartingPort    = 2000
```

Comment:
change the default port to the one you confirm it works

Reply:
Modified the port as the README file mentioned.